### ISSUE TITLE
Potential changes for global boston.gov / global styles search

### ISSUE THREAD

**Issue by reillyzlab** [CityOfBoston/boston.gov-d8#2065]
Creating this ticket for knowledge sharing/later use:

All properties using the global header style take users to boston.gov search, so search discussions are based around Drupal/Boston.gov. 

1. A preliminary A/B test/experiment with the current boston.gov homepages versus a hero image with a large search bar was run the first half of 2019. When this was run the only search that existed was a magnifying glass icon [pressed and opens a search bar.]
- Analytics from this experiment can be found here: https://analytics.google.com/analytics/web/#/dashboard/34PQSRp1SymF2SGzpr0DzA/a99773468w108307691p112911200/_u.date00=20190101&_u.date01=20190506/ [qualitative take away - you add a search bar, people will use it]
- Designs from this experiment for the hero with search can be found here: [@sebastianebarb or @mmcgowanBOS can you add a link here]

Future considerations/options:
- Implement the hero with search bar with testing information we have now.
- Run an experiment in which there is a small search bar that people can type into in the top header/navigations. Further considerations need to be done to figure out how this works on mobile. One suggestion found in the research below is to add a search bar within the hamburger menu, which could be helpful for mobile.
- Consider both of these options.
 
 
2. Swiftype has a module built in to allow search recommendations to users. Consider exploring and implementing: https://app.swiftype.com/engines/boston-dot-gov/engagement_modules
 
 
3. A point of note: None of the 'in page searches' on boston.gov are tied to Swiftype. They are all tied to Drupal views.
 
 
Additional resource/research done a few years ago that includes search findings and recommendations/ideas: https://docs.google.com/document/d/1shfyPLlGW2USAqdWNyGVe6AfZXivpGRE9Lm24kFAKoY/edit

**Comment by duffy-james** [comment]
We've had some informal discussions about implementing a full search bar in the hero, @rachelsater. @mmcgowanBOS had one mocked up and on the site a few years back.

I'd be up for bringing that back, but I got some pushback in the last meeting where we discussed this. It might require a discussion with the larger group, but I do like having that search up front, similar to [mass.gov.](https://www.mass.gov/)
